src/components/userProfile/UserProfileButton.tsx

```diff
@@ -2,32 +2,39 @@ import { Box, Button } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
 
 const UserProfileButton = ({
-  isFriend = false,
   skip,
   beFriend,
   startChat,
 }: {
-  isFriend: boolean
-  skip: () => void
-  beFriend: () => void
-  startChat: () => void
+  skip?: () => void
+  beFriend?: () => void
+  startChat?: () => void
 }) => {
   const { classes } = useStyles()
-  return isFriend ? (
-    <Box className={classes.buttonSection}>
-      <Button className={classes.whiteButton} onClick={startChat}>
-        Start chat
-      </Button>
-    </Box>
-  ) : (
-    <Box className={classes.buttonSection}>
-      <Button className={classes.whiteButton} onClick={skip}>
-        Skip
-      </Button>
-      <Button className={classes.orangeButton} onClick={beFriend}>
-        Be friend
-      </Button>
-    </Box>
+  return (
+    <>
+      {startChat && (
+        <Box className={classes.buttonSection}>
+          <Button className={classes.whiteButton} onClick={startChat}>
+            Start chat
+          </Button>
+        </Box>
+      )}
+      {(skip || beFriend) && (
```

Comment:
Don't you think this check is redundant? You already have separate check for either skip and beFriend inside this construction. Or did you write it for better readability?

Reply:
This check was necessary for `<Box className={classes.buttonSection}>`. But now, I see, that I can simplify the code. I made some changes.